flow/src/main/java/com/sequenceiq/cloudbreak/service/ha/HeartbeatService.java

```diff
@@ -119,12 +119,14 @@ public void heartbeat() {
     @Scheduled(initialDelay = 35000L, fixedDelay = 30000L)
     public void scheduledFlowDistribution() {
         if (shouldRun()) {
```

Comment:
This method says nothing about when it should run. Can you please rename it or just inline the underlying implementation which is so simple:

```java
nodeConfig.isNodeIdSpecified()
```